src/api/ui/views/vanilla-table.tsx

```diff
@@ -60,11 +66,24 @@ export function VanillaTable<T>(props: VanillaTableProps<T>) {
 
     const totalElements = useMemo(() => Groupings.count(props.rows), [props.rows]);
     const paging = useDatacorePaging({ initialPage: 0, paging: props.paging, elements: totalElements });
+    const tableRef = useRef<HTMLTableElement>(null);
 
     const pagedRows = useMemo(() => {
-        if (paging.enabled)
+        if (paging.enabled) {
```

Comment:
I don't recommend doing this inside a `useMemo` call - memo calls having side effects can be confusing. This is a case where a separate `useEffect` call which depends on `paging.page` and `paging.pageSize` is probably correct. You probably also don't want this logic to run whenever `props.rows` changes since that will happen whenever a value changes.